include/bluetooth/hci.h

```diff
@@ -626,6 +626,86 @@ struct bt_hci_rp_read_bd_addr {
 	bt_addr_t bdaddr;
 } __packed;
 
+/* logic transport types used when reading codecs and controller delay */
+#define BT_HCI_LOGICAL_TRANSPORT_TYPE_BREDR_ACL 0
+#define BT_HCI_LOGICAL_TRANSPORT_TYPE_BREDR_SCO 1
+#define BT_HCI_LOGICAL_TRANSPORT_TYPE_LE_CIS    2
+#define BT_HCI_LOGICAL_TRANSPORT_TYPE_LE_BIS    3
+
+/* audio datapath directions */
+#define BT_HCI_DATAPATH_DIR_HOST_TO_CTLR 0
+#define BT_HCI_DATAPATH_DIR_CTLR_TO_HOST 1
+
+/* coding format assigned numbers, used for codec IDs */
+#define BT_HCI_CODING_FORMAT_ULAW_LOG    0x00
+#define BT_HCI_CODING_FORMAT_ALAW_LOG    0x01
+#define BT_HCI_CODING_FORMAT_CVSD        0x02
+#define BT_HCI_CODING_FORMAT_TRANSPARENT 0x03
+#define BT_HCI_CODING_FORMAT_LINEAR_PCM  0x04
+#define BT_HCI_CODING_FORMAT_MSBC        0x05
+#define BT_HCI_CODING_FORMAT_VS          0xFF
+
+
+#define BT_HCI_OP_READ_CODECS                   BT_OP(BT_OGF_INFO, 0x000b)
+struct bt_hci_std_codec_info {
+	uint8_t codec_id;
+} __packed;
+struct bt_hci_vs_codec_info {
+	uint16_t company_id;
+	uint16_t codec_id;
+} __packed;
+struct bt_hci_rp_read_codecs {
+	uint8_t status;
+	/* other fields filled in dynamically */
+} __packed;
+
+#define BT_HCI_OP_READ_CODECS_V2                BT_OP(BT_OGF_INFO, 0x000d)
+struct bt_hci_std_codec_info_v2 {
+	uint8_t codec_id;
+	uint8_t transport;
+} __packed;
+struct bt_hci_vs_codec_info_v2 {
+	uint16_t company_id;
+	uint16_t codec_id;
+	uint8_t transport;
+} __packed;
+struct bt_hci_rp_read_codecs_v2 {
+	uint8_t status;
+	/* other fields filled in dynamically */
+} __packed;
+
+struct bt_hci_cp_codec_id {
+	uint8_t coding_format;
+	uint16_t company_id;
+	uint16_t vs_codec_id;
+} __packed;
+
+#define BT_HCI_OP_READ_CODEC_CAPABS             BT_OP(BT_OGF_INFO, 0x000e)
+struct bt_hci_cp_read_codec_capabs {
+	struct bt_hci_cp_codec_id codec_id;
+	uint8_t ltt;
+	uint8_t dir;
```

Comment:
```suggestion
	uint8_t transport_type;
	uint8_t direction;
```

Reply:
I actually requested to use `dir` instead of `direction`, because I thought I'd keep the same level of info while being shorter, and also better match the `BT_HCI_DATAPATH_DIR_*` defines.

Reply:
Updated to use `transport` and `direction` to be more explicit